deliveryconfig.go

```diff
@@ -889,13 +874,9 @@ func (p *DeliveryConfigProcessor) Delete(cli *Client) error {
 
 // ValidationErrorDetail is the structure of the document from /managed/delivery-configs/validate API
 type ValidationErrorDetail struct {
-	Error    string `json:"error"`
-	Location struct {
-		Column int `json:"column"`
-		Line   int `json:"line"`
-	} `json:"location"`
-	Message        string `json:"message"`
-	PathExpression string `json:"pathExpression"`
+	Error   string  `json:"error"`
+	Status  float32 `json:"status"`
```

Comment:
What information does "Status" encode?

Reply:
The status code (400). It tried using `int` but it failed.
Anyway, I'm not using it so I can probably just remove it.

Reply:
Very odd, I would expect `int` to work, do you have an example payload that you tried to parse?  Capturing the http status seems like a good idea generically, but the float is definitely weird for that.

Reply:
I changed the status to an int, probably something else was broken when I tried it before